README.md

```diff
@@ -1,3 +1,8 @@
+# ⚠️ DISCLAIMER ⚠️
+## This is an internal project. It is only meant to be used by [asset-compute-commons](https://github.com/adobe/asset-compute-commons/blob/master/package.json#L12)
+For custom workers and other components needing to use I/O Events, please use the [aio-lib-events](https://github.com/adobe/aio-lib-events) library.
```

Comment:
I would not mention custom workers at all? Might just be confusing as they are using it indirectly today (for the asset compute events).

Reply:
ACK

Reply:
ACK